### PR TITLE
docs(Foundation/Grid): Updated documentation template

### PR DESCRIPTION
**Pull request by MarcosViniciusPC**
## Description
https://jirasoftware.catho.com.br/browse/QTM-355

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Code review
- [ ] Documentation (yarn storybook)

### CODE REVIEW

File: stories/Grid/Grid.story.mdx [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+The `<Col>` component accepts this props for breakpoints:
+
+<ul>
+  <li>xsmall</li>
+  <li>small</li>
+  <li>medium</li>
+  <li>large</li>
+</ul>
+
+#### Example: Stacked-to-horizontal
```
Comment (toncabral88) [inline stories/Grid/Grid.story.mdx:93]: How about:

```suggestion
## Examples

#### Stacked-to-horizontal
```

**Review by toncabral88 — APPROVED** [review]
I left some suggestion, but everything's ok!

[x] Documentation
[x] Code review

File: stories/Grid/Grid.jsx [hunk 2]
```diff
@@ -1,255 +1,149 @@
...
-        <td colSpan="1" rowSpan="1">
-          <p>large</p>
-        </td>
-        <td colSpan="1" rowSpan="1">
-          <p>12</p>
-        </td>
-        <td colSpan="1" rowSpan="1">
-          <p>24</p>
-        </td>
+        {Object.keys(tableHeadRow).map(key => (
```
Comment (rapahaeru) [inline stories/Grid/Grid.jsx:129]: Wow, Nice!

**Review by rapahaeru — DISMISSED** [review]
All good to me!
I Agree with the @toncabral88 apointments.

File: stories/Grid/Grid.jsx [hunk 3]
```diff
@@ -1,255 +1,149 @@
...
+  margin-bottom: 40px;
+  th,
+  td {
+    padding: 24px 8px;
+    text-align: left;
+  }
+  tr:nth-child(even) {
+    background-color: #f6f8fa;
+  }
+`;
```
[the remark this answers is not in the document]
Comment (toncabral88) [reply stories/Grid/Grid.jsx:23]: Currently this component already exists on master. I think is better rebase this branch and use it to keep standards on project.

File: stories/Grid/Grid.story.mdx (hunk 1 above)
Comment (MarcosViniciusPC) [reply stories/Grid/Grid.story.mdx:93]: Good

File: stories/Grid/Grid.story.mdx [hunk 4]
```diff
@@ -0,0 +1,232 @@
...
+    definitions to each resolution.
+</StyledHeader>
+
+## Importing Grid
+
+```js
+import { Container, Row, Col, Hide } from '@catho/quantum';
+```
+
+### Usage
```
[the remark this answers is not in the document]
Comment (MarcosViniciusPC) [reply stories/Grid/Grid.story.mdx:34]: Good

[the remark this answers is not in the document]
Comment (MarcosViniciusPC) [reply stories/Grid/Grid.story.mdx:34]: But doing this will leave the default used in Colors

**Pull request merged**